bootstrap/managers.go

```diff
@@ -29,8 +28,8 @@ import (
 	codeManagers "stash.kopano.io/kc/konnect/oidc/code/managers"
 )
 
-func newManagers(ctx context.Context, bs *bootstrap) (*managers.Managers, error) {
-	logger := bs.cfg.Logger
+func NewManagers(ctx context.Context, bs *bootstrap) (*managers.Managers, error) {
```

Comment:
Makes no sense to be public, since it requires a private type parameter. Please make private.